### ISSUE TITLE
Add code testing structure for the frontend code (public/js)

### ISSUE THREAD

**Issue by andreban** [GoogleChrome/gulliver#372]
Code in `public/js` is not playing well with our testing configuration, as it uses the ES6 import syntax and we get something like following:

```
gulliver/test/js/event-target.js:18
import EventTarget from '../../public/js/event-target';
^^^^^^
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at /usr/local/lib/node_modules/mocha/lib/mocha.js:222:27
    at Array.forEach (native)
    at Mocha.loadFiles (/usr/local/lib/node_modules/mocha/lib/mocha.js:219:14)
    at Mocha.run (/usr/local/lib/node_modules/mocha/lib/mocha.js:487:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/mocha/bin/_mocha:459:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149
```

**Comment by andreban** [comment]
This seems to work:

```shell
mocha --compilers js:babel-core/register test/js/*.js
```

But needs a different `.babelrc` config. Looking on how to make this run with a different than the one used by rollup.

```js
{
  "presets": ["es2015"]
}
```

More info here: http://www.pauleveritt.org/articles/pylyglot/es6_imports/

**Comment by andreban** [comment]
Looks like the solution is to use a config for babel like this:

```
{
  "env": {
    "test": {
      "presets": ["es2015"]
    },
    "default": {
      "presets": [
        [
          "es2015",
          {
            "modules": false
          }
        ]
      ],
      "plugins": ["external-helpers"]
    }
  }
}
```
And then run tests setting the BABEL_ENV variable:
`BABEL_ENV=test mocha --compilers js:babel-core/register ./test/js/*.js``

Trying to make this play along with npm scripts in package.json

**Comment by andreban** [comment]
Closed  by #376
